js/main.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-restricted-globals */
```

Comment:
- [ ] [OPTIONAL] Kindly remove disabled linters lines in order to make your code cleaner.